9.secretstore/node/components/statestore.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: dapr.io/v1alpha1
```

Comment:
Can delete this file.

Reply:
Removed. Product of auto-creating components folder!